eng/common/sdk-task.ps1

```diff
@@ -64,7 +64,7 @@ try {
       $GlobalJson.tools | Add-Member -Name "vs" -Value (ConvertFrom-Json "{ `"version`": `"16.5`" }") -MemberType NoteProperty
     }
     if( -not ($GlobalJson.tools.PSObject.Properties.Name -match "xcopy-msbuild" )) {
-      $GlobalJson.tools | Add-Member -Name "xcopy-msbuild" -Value "17.2.1" -MemberType NoteProperty
```

Comment:
Is it possible ( and a good idea? ) to automate this version bumping?

Reply:
Ugh, looks like this was a bad merge or something, let me rebase it out.

Reply:
Sorry for the confusion, what's actually happening is that #28591 was an automated update with some test failures.  It turned out there were some breaking changes which the tests were catching, so I created a separate PR starting with the changes from the automated one to fix the regressions.